xc/common/utils/prjxray_create_synth_tiles.py

```diff
@@ -39,11 +36,116 @@ def map_tile_to_vpr_coord(conn, tile):
     return grid_x, grid_y
 
 
+def in_roi(roi, x, y):
```

Comment:
Docstring and some doctests would be good here

Reply:
I was actually able to just remove this function.  I think the other functions I added would be very challenging to write a doctest for given their reliance on the connection database.

Reply:
> I think the other functions I added would be very challenging to write a doctest for given their reliance on the connection database.

Yep, which is why I didn't mention them